x-pack/plugins/watcher/public/application/sections/watch_edit/components/watch_edit.tsx

```diff
@@ -115,7 +116,7 @@ export const WatchEdit = ({
           const loadedWatch = await loadWatch(id);
           dispatch({ command: 'setWatch', payload: loadedWatch });
         } catch (error) {
-          dispatch({ command: 'setError', payload: error });
+          dispatch({ command: 'setError', payload: error.body });
```

Comment:
This addresses a bug I noticed while testing - the error handling wasn't working as expected here, as `getPageErrorCode(loadError)` expects a `statusCode`.